packages/space-opera/src/components/camera_settings/components/fov_limits.ts

```diff
@@ -29,37 +32,19 @@ export const DEFAULT_MIN_FOV = 10;
 /** Default maximum FOV angle (degrees) */
 export const DEFAULT_MAX_FOV = 90;
 
-/** Dispatch change to maximum FOV */
-export const dispatchFovLimits = registerStateMutator(
-    'SET_CAMERA_FOV_LIMITS', (state, fovLimitsDeg?: Limits) => {
-      if (!fovLimitsDeg) {
-        throw new Error('No valid FOV limit given');
-      }
-      if (fovLimitsDeg === state.camera.fovLimitsDeg) {
-        throw new Error(
-            'Do not edit fovLimitsDeg in place. You passed in the same object');
-      }
-
-      state.camera = {
-        ...state.camera,
-        fovLimitsDeg,
-      };
-    });
-
-
 /** The Camera Settings panel. */
 @customElement('me-camera-fov-limits')
 export class FovLimits extends LimitsBase {
   @internalProperty() fovLimitsDeg?: Limits;
-  @internalProperty() currentCamera?: Camera;
+  @internalProperty() isDirtyCamera: boolean = false;
 
   stateChanged(state: State) {
-    this.fovLimitsDeg = state.camera.fovLimitsDeg;
-    this.currentCamera = state.currentCamera;
+    this.fovLimitsDeg = getCamera(state).fovLimitsDeg;
+    this.isDirtyCamera = getIsDirtyCamera(state);
```

Comment:
this is used within camera_settings/components/ to force an update to the state, so the corresponding numbers are adjusted.